animation_test.go

```diff
@@ -0,0 +1,17 @@
+package fyne
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAnimationLinear(t *testing.T) {
+	assert.Equal(t, float32(0.25), animationLinear(0.25))
+}
+
+func TestAnimationEaseInOut(t *testing.T) {
```

Comment:
Missing `TestAnimationEaseIn` and `TestAnimationEaseOut`

Reply:
thanks